sample_config.py

```diff
@@ -14,3 +14,4 @@
 HEARTBEAT_ENDPOINT = "/api/v1/heartbeat"
```

Comment:
Since you have renamed this file, all imports depending on config are breaking. i.e. `from config import *`. Which is fine I think. Could you add an explanation about this in the ReadMe? The courses are supposed to create their own config and name it `config.py`.

Alternatively (and preferably), I would not rename this. Let `config.py` be with all the default values. `config.py` contains stuff that a user should not be modifying or fiddling with. Like endpoints. That a developer level config. So you could mention in the ReadMe that `config.py` is initialized with default values, and they could change stuff if they like (like the heartbeat interval, etc).

Reply:
👍